pages/_app.js

```diff
@@ -2,12 +2,20 @@ import GlobalStyle from "../styles";
 import Head from "next/head";
 import Footer from "../components/Footer";
 import { useSWRConfig, SWRConfig } from "swr";
+import { useState } from "react";
 
 export default function App({ Component, pageProps }) {
   const fetcher = (url) => fetch(url).then((response) => response.json());
```

Comment:
It's good practice to declare the **fetcher** function outside of the `App` component.
```suggestion

const fetcher = (url) => fetch(url).then((response) => response.json());

export default function App({ Component, pageProps }) {
```